extensions/strategies/trendline/strategy.js

```diff
@@ -1,113 +1,66 @@
-let math = require('mathjs')
-  , trend = require('trend')
-  , z = require('zero-fill')
-  , n = require('numbro')
+var z = require('zero-fill')
   , stats = require('stats-lite')
+  , math = require('mathjs')
   , ema = require('../../../lib/ema')
   , Phenotypes = require('../../../lib/phenotype')
-var oldgrowth = 1
+  , regression = require('regression')
+  , trend = require('trend')
+  , n = require('numbro')
 
 module.exports = {
   name: 'trendline',
-  description: 'Calculate a trendline and trade when trend is positive vs negative.',
-
+  description: 'Buy on positive trendline above 1.1 Make sure to set --days when backtesting period.',
   getOptions: function () {
-    this.option('period', 'period length', String, '30s')
-    this.option('periodLength', 'period length', String, '30s')
-    this.option('lastpoints', 'Number of trades for short trend average', Number, 100)
-    this.option('avgpoints', 'Number of trades for long trend average', Number, 1000)
-    this.option('lastpoints2', 'Number of trades for short trend average', Number, 10)
-    this.option('avgpoints2', 'Number of trades for long trend average', Number, 100)
-    this.option('min_periods', 'Basically avgpoints + a BUNCH of more preroll periods for anything less than 5s period', Number, 15000)
-    this.option('markup_sell_pct', 'test', Number, 0)
-    this.option('markdown_buy_pct', 'test', Number, 0)
+    this.option('period', 'period length, set poll trades to 100ms, poll order 1000ms. Same as --period_length', String, '1h')
+    this.option('period_length', 'period length, set poll trades to 100ms, poll order 1000ms. Same as --period', String, '1h')
+    this.option('avgpoints', 'Trades for array 2 to be calculated stddev and mean from', Number, 100)
+    this.option('lastpoints', 'Trades for array 2 to be calculated stddev and mean from', Number, 5)
+    this.option('min_periods', 'min_periods', Number, 150)
+    this.option('order_adjust_time', 'Order Adjust Time', Number, 30000)
```

Comment:
as this is an internal engine configuration i would not give this to the strategy. it does not depend on that, and i would not advise to darwin on that, since the simulation does not have a real "timeout" and market change implementation. it will result in unkown behavior in real trade scenario.